src/compute-pr-actions.ts

```diff
@@ -121,6 +124,7 @@ export function process(info: PrInfo | BotEnsureRemovedFromProject | BotNoPackag
     context.labels["New Definition"] = info.anyPackageIsNew;
     context.labels["Author is Owner"] = info.authorIsOwner;
     context.labels["Merge:Auto"] = canBeMergedNow(info);
+    context.isClassedAsReadyForMerge = canBeMergedNow(info);

```

Comment:
so the problem was that `canBeMergedNow` wasn't persisted before?

Reply:
effectively it wasn't being passed down to the 'commenting' stage and couldn't be used to show/remove that message